src/licensedcode/data/rules/gpl-2.0_1436.RULE

```diff
@@ -0,0 +1,12 @@
+---
+license_expression: gpl-2.0
+is_license_notice: yes
+---
+
+* Released under the terms of the {{GNU GPL v2.0.}}
```

Comment:
Why not just have only "Released under the terms of the {{GNU GPL v2.0.}}" as a rule?

Reply:
We are having a low score here otherwise as GNU is present and considered as license text? Maybe we should make `_GNU_SOURCE` a false positive?